git-extra/astextplain

```diff
@@ -8,6 +8,9 @@ fi
 
 # XXX output encoding (UTF-8) hardcoded
 case "$1" in
+	*.ods | *.ODS | *odf |*.ODF | *odt | *.ODT)
+		odf2txt "$1" || cat "$1"
```

Comment:
Pacman is not only for Arch Linux. Yes, it was born there, but the [MSYS2 project](https://msys2.github.io/) uses it, too. And Git for Windows' SDK is a custom version of MSYS2. Git for Windows is essentially a subset of the custom MSYS2 development environment.

You can learn much more about this here: https://github.com/git-for-windows/git/wiki/Package-management

As to an example of a recently-added MINGW package, how about this: https://github.com/git-for-windows/MINGW-packages/commit/898f5ed4bd2c6f5addd60aabb977051d97d0350f